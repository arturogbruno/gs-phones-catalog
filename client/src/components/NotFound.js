import React from "react";
import { Link } from "react-router-dom";

const NotFound = () => {
    return(
        <div className="NotFound">
            <img src="/images/notfound.png" alt="not found"/>
            <h1>Ups... we can't find the page you were looking for.</h1>
            <Link to="/">Take me back!</Link>
        </div>
    )
}

export default NotFound;