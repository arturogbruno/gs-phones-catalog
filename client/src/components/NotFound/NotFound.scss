@import "../../styles/mixins.scss";

.NotFound {
    background-color: rgba(255,255,255,0.6);
    width: 80%;
    box-sizing: border-box;
    padding: 30px;
    display: flex;
    flex-direction: column;
    align-items: center;

    img {
        width: 80%;
        margin-bottom: 50px;

        @include md {
            width: 50%;
        }
    }

    h1 {
        font-size: 1.5em;
        text-align: center;
    }

    button {
        margin-top: 30px;
        border: 3px solid rgb(3, 185, 153);
        outline: none;
        background-color: transparent;
        padding: 10px 20px;
        transition-duration: 0.4s;
          
        &:hover {
            background-color: rgb(3, 185, 153);
            cursor: pointer;
        }
        
        a {
            text-decoration: none;
            font-size: 1.5em;
            color: rgb(35, 35, 35);

        }
    }

    @include lg {
        width: 50%;
    }
}
