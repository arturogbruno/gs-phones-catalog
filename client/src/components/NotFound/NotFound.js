import React from "react";
import { Link } from "react-router-dom";
import "./NotFound.scss";

const NotFound = () => {
    return(
        <div className="NotFound">
            <img src="/images/notfound.png" alt="not found"/>
            <h1>Ups... we can't find the page you were looking for.</h1>
            <button><Link to="/">Take me back!</Link></button>
        </div>
    )
}

export default NotFound;