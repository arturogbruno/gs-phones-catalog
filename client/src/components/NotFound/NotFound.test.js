import React from "react";
import { shallow } from "enzyme";
import NotFound from "./NotFound";

describe("testing NotFound component", () => {
    it("should render NotFound component", () => {
        expect(shallow(<NotFound />)).toMatchSnapshot();
    });
});