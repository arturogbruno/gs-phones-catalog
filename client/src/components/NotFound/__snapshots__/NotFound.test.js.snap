// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing NotFound component should render NotFound component 1`] = `
<div
  className="NotFound"
>
  <img
    alt="not found"
    src="/images/notfound.png"
  />
  <h1>
    Ups... we can't find the page you were looking for.
  </h1>
  <button>
    <Link
      to="/"
    >
      Take me back!
    </Link>
  </button>
</div>
`;
